### ISSUE TITLE
CPU high

### ISSUE THREAD

**Issue by RockSteadyBeat** [SphtKr/homebridge-itunes#5]
Hello folks,
since i have update iTunes to 12.4.0.119 i can´t start the homebridge-itunes plugin anymore...
the cpu-counter of the task "homebridge" get´s up to 99,9% and that´s it...

any ideas?

Sorry about my english, i´m not a native speaker... ;-)

edit:

here´s my config.json

{
    "bridge": {
        "name": "Homebridge@iTunes",
        "username": "CC:22:3D:E3:CE:49",
        "port": 51826,
        "pin": "031-45-155"
    },

```
"platforms": [{
    "platform": "homebridge-fhem.FHEM",
    "name": "FHEM",
    "server": "192.168.253.253",
    "port": "8083",
    "filter": "room=homekit_devices"
}, {
    "platform": "iTunes"
}],

"accessories": [

]
```

}

**Comment by SphtKr** [comment]
Hmm... Do you happen to notice a difference if iTunes is already running when you start Homebridge? I know that doesn't work well when Homebridge starts on a script, but it would help narrow it down.

**Comment by RockSteadyBeat** [comment]
Nope, i try it in two way´s, if iTunes is running before starting homebridge it´s the same like it won´t run at start...

here´s some log at start:
[2016-06-23 12:36:51] Loaded plugin: homebridge-fhem
homebridge API version: 2
[2016-06-23 12:36:51] Registering platform 'homebridge-fhem.FHEM'
[2016-06-23 12:36:51] ---
[2016-06-23 12:36:51] Loaded plugin: homebridge-itunes
[2016-06-23 12:36:51] Registering platform 'homebridge-itunes.iTunes'
[2016-06-23 12:36:51] ---
[2016-06-23 12:36:51] Loaded config.json with 0 accessories and 2 platforms.
[2016-06-23 12:36:51] ---
[2016-06-23 12:36:51] Loading 2 platforms...
[2016-06-23 12:36:51] [FHEM] Initializing homebridge-fhem.FHEM platform...
starting longpoll: http://192.168.253.253:8083/fhem.pl?XHR=1&inform=type=status;addglobal=1;filter=.*;since=null;fmt=JSON&timestamp=1466678213486

i can't read that homebridge-itunes initialize...???

**Comment by fugazzy** [comment]
Hello I have the exact problem:
I am trying to install my second homebridge now on my Mac for iTunes (I have one on rpi2). Node 4.4.7 At the moment without any plugins, because I can't get it to pair with iOS. 
I just discovered that homebridge  is using 100% CPU on my system, that seems to be the cause. I already tried different node versions and a complete new install.  Debug-Mode does not show anything. Any hints?
The iTunes-plugin seems to be the cause
Even if it is not configured in the config.json (only installed) , no matter if iTunes 12.4.1.6 is running or not
username of my 2nd home bridge is changed and it also shows up on iOS
